src/main/webapp/app/exam/participate/exercises/file-upload/file-upload-exam-submission.component.html

```diff
@@ -1,7 +1,7 @@
 <!--resizable container-->
 <jhi-resizeable-container *ngIf="exercise">
     <!--region Left Panel-->
-    <span class="exercise-title" left-header *ngIf="exercise.title">{{ exercise.title }}</span>
+    <span class="exercise-title" left-header *ngIf="exercise?.exerciseGroup?.title">{{ exercise?.exerciseGroup?.title }}</span>
```

Comment:
Some placeholder like the dash would probably be better than leaving it out completely. (Also applies to other occurrences)
```suggestion
    <span class="exercise-title" left-header>{{ exercise?.exerciseGroup?.title ?? - }}</span>
```